modules/ROOT/pages/app-to-app-requests.adoc

```diff
@@ -0,0 +1,23 @@
+= Access an Application in the Same Cluster
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In an Anypoint Runtime Fabric cluster, every application replica exposes an endpoint inside the cluster. When you enable inbound traffic, that endpoint is exposed externally. Refer to xref:enable-inbound-traffic.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric] for additional information about servicing external requests and configuring the load balancer. 
+
+To send a request from one application to another application inside the cluster, use the application name and environment ID. 
+
+Every application has a DNS record in the cluster internal domain. For example, the DNS record for the application `hello-world` in the  `2abc2ce6-cce6-463b-afaa-a1979e3xxxxx` environment is:
+```
+hello-world.2abc2ce6-cce6-463b-afaa-a1979e3xxxxx.svc.cluster.local
+```
+Because application replicas first search DNS records in the same domain before searching the parent domain, you need only the application name and environment ID to send a request to another application in the same cluster. If both applications are in the same namespace, you need only the application name:
+```
+http://hello-world:8081/api
```

Comment:
It's not clear to me how this syntax is "only the application name" of the previous syntax: where do 8081 and api come into it?